templates/rate-plan-detail.html.twig

```diff
@@ -6,7 +6,12 @@
  * Variables:
  * - detail
  *   - id:
```

Comment:
`id:` line has no docs.  Is `detail` the rate plan entity?   If so, maybe we should just call it `rate_plan`?

Reply:
I cleaned up the docs for this file.